.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,100 @@
+name: Slow tests (on push)
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
enhancement: do not run the test if we modify only md files

Reply:
use paths: https://github.com/huggingface/optimum-neuron/blob/main/.github/workflows/test_inf1.yml